engine/src/test/java/org/camunda/bpm/engine/test/concurrency/CompetingMessageCorrelationTest.java

```diff
@@ -192,6 +204,7 @@ public void testConcurrentExclusiveCorrelationToDifferentExecutions() throws Int
     // thread two correlates and acquires the exclusive lock on the event subscription of instance2
     // depending on the database and locking used, this may block thread2
     thread2.makeContinue();
+    Thread.sleep(3000); // wait a bit until t2 is blocked during the flush
```

Comment:
Yes, this was added before we concluded that Exclusive Message Correlations can't be used on CRDB. Since the test is excluded for CRDB, we can revert the test to it's original setup.